node/tests/initialization_test.rs

```diff
@@ -60,20 +60,14 @@ fn initialization_sequence_integration() {
         true,
     );
     let mut initialization_client = UiConnection::new(daemon_port, NODE_UI_PROTOCOL);
-    let data_directory = std::env::current_dir()
-        .unwrap()
-        .join("generated")
-        .join("test")
-        .join("initialization_sequence_integration")
-        .to_string_lossy()
-        .to_string();
+    let data_directory = node_home_directory("integration", "initialization_sequence_integration");
```

Comment:
If you use the appropriate ensure_xxx() function, you shouldn't need the fs::create_dir_all() below anymore.